code/modules/telesci/gps.dm

```diff
@@ -125,6 +129,9 @@ var/list/SPS_list = list()
 	if(href_list["toggle_refresh"])
 		autorefreshing = !autorefreshing
 		return TRUE
+	if(href_list["toggle_transmit"])
+		transmitting = !transmitting
```

Comment:
This should probably be set to fail if ``emped`` is ``TRUE``. Otherwise you can leave the interface open and hit the button to instantly get your GPS or SPS back to transmitting after you get EMP'd.